Source/SG/SG.cs

```diff
@@ -172,10 +174,31 @@ public static void DownloadWaveform(NIRfsg rfsgHandle, ref Waveform waveform)
             NIRfsgPlayback.StoreWaveformRuntimeScaling(rfsgPtr, waveform.WaveformName, -1.5);
             NIRfsgPlayback.StoreWaveformRFBlankingEnabled(rfsgPtr, waveform.WaveformName, false);
```

Comment:
this seems counter-productive. why set the burst start and stop locations in RFSG playback if you are going to hardcode blanking to false?

Reply:
Even though blanking is disabled, we still use the database features of RFSGPlayback in the test sequences. For example, when someone is sweeping waveforms in TestStand, I need to retrieve the waveform's properties from the database dynamically. The burst start/stop locations are leveraged for calculating the total length of the waveform; hence, they need to be stored in the database either way for later retrieval.

Reply:
Okay fair enough